src/lib/snack-bar/simple-snack-bar.scss

```diff
@@ -16,6 +16,14 @@ $mat-snack-bar-button-margin: 48px !default;
   background: none;
   flex-shrink: 0;
   margin-left: $mat-snack-bar-button-margin;
+  line-height: 20px;
+
+  .mat-button-ripple,
+  .mat-button-focus-overlay {
+    height: 36px;
```

Comment:
Why does this have to override the `height` and `top`? AFAIK the overlay and ripple are supposed to be `width: 100%` and `height: 100%` of the button?

Reply:
The issue that I am seeing is that the size of the button is supposed to fall outside  of the content box of the `mat-simple-snackbar`.

The snackbar provides `20px` of space in a single line, but the button is supposed to be `36px`.  The "best" solution I think would actually be to move the padding-like spacing from the container to the `mat-simple-snackbar`.   Thoughts?

Reply:
You should be able to work around that by using a negative margin. E.g. if you remove the size and position overrides of the ripple and focus overlay, as well as the `line-height: 20px` from the button and add something like `margin: -14px -24px -14px 48px` you can get the button to overlap the container padding. You might have to tweak the spacing a bit.